src/components/PaymentRequestRow/PaymentRequestRow.tsx

```diff
@@ -39,6 +56,8 @@ const Row = ({ status, createdAt, contact, amount, currency, tags, onClick }: Pa
         {tags.map((tag) => (
           <Chip label={tag} />
         ))}
+
+        <PaymentRequestActionMenu onDuplicate={onDuplicate} onCopyLink={onCopyLink} onDelete={onDelete} />
```

Comment:
In my opinion this should go in a different `<td>` (you will need to add a new `<th>` in the Row to achieve this too)